pyomo/util/tests/test_calc_var_value.py

```diff
@@ -32,36 +33,57 @@ def test_initialize_value(self):
         m.x.set_value(None)
         calculate_variable_from_constraint(m.x, m.c)
         self.assertEqual(value(m.x), 5)
+        m.x.set_value(None)
+        calculate_variable_from_constraint(m.x, m.c, diff_mode=differentiate.Modes.reverse_numeric)
+        self.assertEqual(value(m.x), 5)
 
```

Comment:
Should we also update these tests for `reverse_symbolic` and explicitly `sympy`?

Reply:
Done!